app/javascript/lib/efile_security_information.js

```diff
@@ -5,17 +5,17 @@ export function getEfileSecurityInformation() {
     let Fingerprint2 = initFingerprint2();
     let CryptoJS = initCryptoJS();
     Fingerprint2.get(function(components) {
-        var stringtohash = components.map(function (pair) { return pair.value }).join('###')
-        let encrypted_device_id =  CryptoJS.SHA1(stringtohash);
-        document.getElementById('ctc_consent_form_device_id').value = encrypted_device_id;
+        let concatenated = components.map(function (pair) { return pair.value }).join('###')
+        let concatenatedAndHashed = CryptoJS.SHA1(concatenated).toUpperCase();
```

Comment:
I renamed this variable because SHA1 isn't an encryption function, but a [hashing function.](https://en.wikipedia.org/wiki/SHA-1)